api/v1alpha1/fenceagentsremediation_types.go

```diff
@@ -74,12 +74,10 @@ type FenceAgentsRemediationStatus struct {
 
 	// Represents the observations of a FenceAgentsRemediation's current state.
 	// Known .status.conditions.type are: "Processing", "FenceAgentActionSucceeded", and "Succeeded".
-	// +patchMergeKey=type
-	// +patchStrategy=merge
 	// +listType=map
 	// +listMapKey=type
 	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="conditions",xDescriptors="urn:alm:descriptor:io.kubernetes.conditions"
-	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
+	Conditions []metav1.Condition `json:"conditions,omitempty" protobuf:"bytes,1,rep,name=conditions"`
```

Comment:
missed in the first review: why the `protobuf` tag?

Reply:
That doc is primarily targeting core k8s types. So in general it's good to follow its recommendations, but not everything applies to CRDs as well 🙂